mod/register.php

```diff
@@ -292,7 +292,9 @@ function register_content(App $a)
 		'$showprivstatement' => Config::get('system', 'tosprivstatement'),
 		'$privstatement' => $tos->privacy_complete,
 		'$baseurl'   => System::baseurl(),
-		'$form_security_token' => get_form_security_token("register")
+		'$form_security_token' => get_form_security_token("register"),
+		'$explicit_content' => Config::get('system', 'explicit_content', False),
```

Comment:
Code Standards: boolean values should be written lowercase except if the prevailing surrounding spelling is different,